composer.json

```diff
@@ -24,7 +24,9 @@
         "drupal/console": "~1.0",
         "drupal/field_group": "~8.0",
         "drupal/state_machine": "~8.0",
-        "drupal/admin_toolbar": "~8.0"
+        "drupal/admin_toolbar": "~8.0",
+        "drupal/address": "~8.0",
+        "drupal/paragraphs": "~8.1"
```

Comment:
Kinda surprised to see Paragraphs here. Its a great module, but complex. Each paragraph is its own entity reference